_schemaV2.graphql

```diff
@@ -22380,27 +22333,23 @@ input ViewingRoomSubsectionInput {
   internalID: ID
 }
 
-# The connection type for ViewingRoom.
+# A connection to a list of items.
 type ViewingRoomsConnection {
   # A list of edges.
   edges: [ViewingRoomsEdge]
-
-  # A list of nodes.
-  nodes: [ViewingRoom]
-  pageCursors: PageCursors
+  pageCursors: PageCursors!
 
   # Information to aid in pagination.
   pageInfo: PageInfo!
   totalCount: Int
-  totalPages: Int
```

Comment:
couldn't find totalPages being used in Force / Eigen, hopefully we can delete this field?

Reply:
`yarn relay` should reveal it